docs/news.rst

```diff
@@ -6,11 +6,185 @@ Release notes
 Scrapy 1.6.0 (unreleased)
 -------------------------
 
-Cleanups
-~~~~~~~~
+Highlights:
+
+* better Windows support;
+* Python 3.7 compatibility;
+* big documentation improvements, including a switch
+  from ``.extract()`` + ``.extract_first()`` API to ``.get()`` + ``.getall()``
```

Comment:
It is a really minor thing, but both here and in references below I would suggest to switch the order of one of the pairs so that there is parallelism between the method references. That is, either:
```suggestion
  from ``.extract()`` + ``.extract_first()`` API to ``.getall()`` + ``.get()``
```
or
```suggestion
  from ``.extract_first()`` + ``.extract()`` API to ``.get()`` + ``.getall()``
```

Reply:
Thanks @Gallaecio, applied! `.extract()` should be replaced with `.get()` for Selector, so it is not truly parallel, but anyways, your suggested order is better.